### PR TITLE
Feature/integration angular

### PR DESCRIPTION
**Pull request by bdruziuk**

### CODE REVIEW

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/rich-text.directive.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import {
```
Comment (Antonytm) [inline astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/rich-text.directive.ts:1]: Please format spacing to follow the project style: 2 spaces.

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/text.directive.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import {
```
Comment (Antonytm) [inline astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/text.directive.ts:1]: Please format spacing to follow the project style: 2 spaces.

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/Styleguide-AngularUsage-Sitecore-Placeholder.astro [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (Antonytm) [inline astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/Styleguide-AngularUsage-Sitecore-Placeholder.astro:1]: As Placeholder with Angular is not implemented, please remove this file from the PR.

**Review by Antonytm — CHANGES_REQUESTED** [review]

File: sitecore/src/Serialization/items/headless-astro/Astro/home/styleguideExtended/Page Data/Styleguide-Angular-Sitecore-Placeholder.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+﻿---
```
[the remark this answers is not in the document]
Comment (bdruziuk) [reply sitecore/src/Serialization/items/headless-astro/Astro/home/styleguideExtended/Page Data/Styleguide-Angular-Sitecore-Placeholder.yml]: done

File: sitecore/src/Serialization/items/samples-astro-renderings/Astro/Integrations/Styleguide-AngularUsage-Sitecore-Placeholder.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+﻿---
```
[the remark this answers is not in the document]
Comment (bdruziuk) [reply sitecore/src/Serialization/items/samples-astro-renderings/Astro/Integrations/Styleguide-AngularUsage-Sitecore-Placeholder.yml]: done

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/rich-text.directive.ts (hunk 1 above)
Comment (bdruziuk) [reply astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/rich-text.directive.ts:1]: done

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/text.directive.ts (hunk 2 above)
Comment (bdruziuk) [reply astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/angular/Standalone/text.directive.ts:1]: done

File: astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/Styleguide-AngularUsage-Sitecore-Placeholder.astro (hunk 3 above)
Comment (bdruziuk) [reply astro-sitecore-jss/packages/astro-sitecore-jss-sample/src/components/integrations/Styleguide-AngularUsage-Sitecore-Placeholder.astro:1]: done

**Review by Antonytm — APPROVED** [review]

**Pull request merged**